tutorbackup/templates/backup/build/backup/backup_local_services.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+mysqldump --all-databases --single-transaction --quick --quote-names --max-allowed-packet=16M \
+  --password=$MYSQL_ROOT_PASSWORD --user=$MYSQL_ROOT_USERNAME --host=$MYSQL_HOST --port=$MYSQL_PORT > /data/mysql_dump.sql
+mongodump --out=/data/mongodb_dump --host=$MONGODB_HOST --port=$MONGODB_PORT
+cp /caddy/Caddyfile /data/Caddyfile
```

Comment:
I don't think this is right. Here's what `/data/caddy` looks like on a Caddy pod that includes certificate data:

```
$ kubectl exec -it caddy-686bb5d4bb-whfnj -- ls /data/caddy -la
total 24
drwx------    6 root     root          4096 Feb  9 14:15 .
drwxr-xr-x    4 root     root          4096 Feb  9 08:24 ..
drwx------    5 root     root          4096 Feb  9 08:31 acme
drwx------    3 root     root          4096 Feb  9 11:00 certificates
drwx------    2 root     root          4096 Feb  9 14:15 locks
drwx------    2 root     root          4096 Mar 23 19:20 ocsp
```
I suppose what's relevant for a backup is that certificate data, not the `Caddyfile` (which Tutor will happily recreate). Or am I mistaken?

Reply:
Yes you are right. I am not familiar with Caddy so I had to do some digging. In my local Tutor deployment these files didn't exist because they are [only written if `ENABLE_HTTPS` and `ENABLE_WEB_PROXY` are set to `True`.](https://github.com/overhangio/tutor/blob/master/tutor/templates/local/docker-compose.prod.yml#L14). I'll change so that we also copy these files if those conditions are met in local. Also will keep this in mind for k8s. 

Is it only certificates that are interesting? what are those other directories?

Reply:
- `acme`: account information for Let's Encrypt
- `ocsp`: OCSP data (primarily certificate revocation details)
- `locks`: lock files (I think these are only used while a certificate renewal is running, and don't need to be backed up)

Reply:
Is it better to copy the whole folder instead of being specific? Maybe there are other things in there that we haven't seen in our setup?

Reply:
I'm actually not sure what would happen if by a freak coincidence, the backup ran in the middle of an upgrade, included the lockfiles in `locks`, was then restored to a new location, and _then_ the next certificate renewal would kick in. Maybe you'd like to use `tar --exclude=locks` (or `--exclude=locks/*`) to include everything *except* the `locks` directory (or its contents).

Reply:
ok I can do that.